### PR TITLE
Update readme with details about archive

### PR DESCRIPTION
**Pull request by smarlowucf**
And clean up option description for archive values in example config.

### CODE REVIEW

**Review by rjschwei — APPROVED** [review]

**Review by KeithMnemonic — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -185,6 +196,41 @@ timestamp that denotes when the usage record was reported and
...
+        },
+        'billed_usage': {
+            '{usage_key}': int
+        },
+        'usage_records': [
+            {
+                '{usage_metric}': int,
+                'reporting_time': 'string(date timestamp)',
+                'base_product': 'cpe:/o:suse:product:v1.2.3'
+            }
```
Comment (rtamalin) [inline README.md:223]: This example kind of suggests that there is only one entry in the usage_records list... Perhaps something like would be a better representation?

```suggestion
            },
            ...
```

File: examples/README.md [hunk 2]
```diff
@@ -31,10 +31,10 @@ reporting_interval:
     Sets the time in seconds when the csp-billing-adapter reports to the CSP API. The values the csp-billing-adapter reports are determined by a combination of the settings for `reporting_api_is_cumulative` and `billing_interval`.
 
 archive_retention_period:
-    Sets the time in months that meterings and usage records are retained in the data archive.
+    Sets the time in billing cycles that meterings and usage records are retained in the data archive. For example when the billing period is is in months a value of 6 will retain 6 months of records. If the value is set to 0 the archive functionality will be disabled.
```
Comment (rtamalin) [inline examples/README.md:34]: Duplicate `is`
```suggestion
    Sets the time in billing cycles that meterings and usage records are retained in the data archive. For example when the billing period is in months a value of 6 will retain 6 months of records. If the value is set to 0 the archive functionality will be disabled.
```

**Review by rtamalin — CHANGES_REQUESTED** [review]
Looks great.

Made one suggestion for better visual conveyance of multiple elements in the usage_records list, and one typo correction.

**Review by KeithMnemonic — APPROVED** [review]
LGTM

**Pull request merged**